src/App.tsx

```diff
@@ -1,24 +1,330 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import * as todoService from './api/todos';
+import { TodoContext } from './todoContext';
+import { TodoList } from './todoList';
 
-const USER_ID = 0;
+const USER_ID = 22;
 
 export const App: React.FC = () => {
+  // #region UseStates
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [titleValue, setTitleValue] = useState('');
+  const [count, setCount] = useState(0);
+  const [selectedFilter, setSelectedFilter] = useState('All');
+  const [error, setError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isSubmitting, setIsSubmiting] = useState(false);
+  const [isDeliting, setIsDeliting] = useState(false);
+  // #endregion
+
+  let errorTimerId: NodeJS.Timeout;
+
+  const errorHandler = (errorPlace: string) => {
+    clearTimeout(errorTimerId);
+
+    switch (errorPlace) {
+      case 'title':
+        setErrorMessage('Title should not be empty');
+        break;
+      case 'add':
+        setErrorMessage('Unable to add a todo');
+        break;
+      case 'update':
+        setErrorMessage('Unable to update a todo');
+        break;
+      case 'delete':
+        setErrorMessage('Unable to delete a todo');
+        break;
+      default:
+    }
+
+    errorTimerId = setTimeout(() => {
+      setError(false);
+    }, 3000);
+  };
+
+  useEffect(() => {
+    todoService.getTodos(USER_ID)
+      .then(data => {
+        setTodos(data);
+        setCount(data.length);
+      })
+      .catch(e => {
+        setError(true);
+        errorHandler('add');
+        throw e;
+      });
+    // eslint-disable-next-line
+  }, []);
+
+  // #region Handlers
+
+  const handleInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitleValue(event.target.value);
+  };
+
+  const filteredTodo = todos.filter(todo => {
```

Comment:
```suggestion
  const filteredTodos = todos.filter(todo => {
```
it's an array of todo**s**